ASSIGMENT/src/pages/dl.astro

```diff
@@ -0,0 +1,271 @@
+---
+import MainLayout from '../layouts/MainLayout.astro';
+import Nav from '../components/Nav.astro';
+---
+
+<head>
+    <link rel="stylesheet" type="text/css" href="/assets/styles/dl.css">
+</head>
+<MainLayout>
+    <nav slot="nav">
+        <Nav />
+    </nav>
+    <section>
+        <label for="documentType" class="dev">Document DL </label>
+        <form id="dlForm">
+            <div id="documentFields">
+                <div class='formContainer' id='dl'>
+                    <label for="documentNumber_dl">DL Number</label>
+                    <input type="text" id="documentNumber_dl" name="documentNumber" required>
+                    <label for="holdingPersonName_dl">Name</label>
+                    <input type="text" id="holdingPersonName_dl" name="holdingPersonName" required>
+                    <label for="DOB_dl">Date of Birth</label>
+                    <input type="date" id="DOB_dl" name="DOB" required>
+                    <label for="DOE_dl">Date of Expiry</label>
+                    <input type="date" id="DOE_dl" name="DOE" required>
+                    <div class="gender-container">
+                        <label for="gender">Gender</label>
+                        <input type="radio" id="gender_male" name="gender" value="male" required checked>
+                        <label for="gender_male">Male</label>
+                        <input type="radio" id="gender_female" name="gender" value="female" required>
+                        <label for="gender_female">Female</label>
+                    </div>
+                </div>
+            </div>
+        </form>
+
+        <div class="button-container">
+            <button type="button" class="resetButton">Reset</button>
+            <button type="submit" class="saveButton">Save</button>
+        </div>
+    </section>
+    <div class="container">
+        <div class="item header">
+            <div>#</div>
+            <div>Document</div>
+            <div>Info</div>
+            <div>Name</div>
+            <div>Gender</div>
+            <div>DOB</div>
+            <div>Actions</div>
+        </div>
+    </div>
+    <div id="canvasContainer" style="display: none;">
+        <canvas id="dlCanvas"></canvas>
+        <button id="downloadButton" style="display: none;">Download Image</button>
+    </div>
+
+    <script>
+        document.addEventListener('DOMContentLoaded', () => {
```

Comment:
```suggestion
        document.addEventListener('DOMContentLoaded', () => {
```

remove this from all paces use `is:inline` directive in script tag. check Astro docs for more information

Reply:
done